### ISSUE TITLE
pdf rendered with errors: "Error: Required "glyf" or "loca" tables are not found"

### ISSUE THREAD

**Issue by mehlis** [mozilla/pdf.js#6007]
http://www.tp-link.com/resources/document/Archer_C7_V2_User_Guide_1910010809.pdf

tested with ff 37.0.2 on linux.

on page 3/4:
```
"PDF 81db909953679b4aa345f95eb9537722 [1.6 Adobe PDF Library 10.0 / Acrobat PDFMaker 10.1 Word 版] (PDF.js: 1.1.124)" viewer.js:6397:0
"Error: Required "glyf" or "loca" tables are not found" pdf.worker.js:243:4
"Font_checkAndRepair@resource://pdf.js/build/pdf.worker.js:17896:11
Font@resource://pdf.js/build/pdf.worker.js:16462:16
PartialEvaluator_translateFont@resource://pdf.js/build/pdf.worker.js:12002:14
PartialEvaluator_loadFont@resource://pdf.js/build/pdf.worker.js:10775:1
PartialEvaluator_handleSetFont@resource://pdf.js/build/pdf.worker.js:10547:14
next@resource://pdf.js/build/pdf.worker.js:10936:22
next/<@resource://pdf.js/build/pdf.worker.js:10941:19
" pdf.worker.js:244:0
"Warning: Unsupported feature "unknown"" pdf.worker.js:235:4
"Warning: Unsupported feature "unknown"" pdf.js:235:4
"Warning: Unsupported feature "font"" pdf.worker.js:235:4
"Warning: Unsupported feature "font"" pdf.js:235:4
"Warning: Error during font loading: Required "glyf" or "loca" tables are not found" pdf.js:235:4
```

**Comment by timvandermeij** [comment]
Possibly a duplicate of #4684.

**Comment by Lord-of-Darkness** [comment]
Developers and leaders of Mozilla, which are shemales ass!

Someone should hang you up in your own balls and let your dog bite his caricature of a cock!

They should then put it long as needed by the throat until he swallows themselves in suicide!

You Mozilla is... Stupid fucking pussies!
.....................................................
You can lick my penis and my golden eggs and kiss my ass son of a bitch with a personality disorder from the depths of the abyss of hell from the abysses of Mozilla!

Fuck you and your mother you autistic son of a bitch, you will lick Putin eggs, fascist fucking faggot!

The shit out of your dog, your cat and your dirty faggot asshole mountain goat!

Suck cocks Mozilla and together go and suck Google ass full of shit!

Shit in your knee you cock suckers! And that damn Australis Chrome clone of shit! Mozilla gay homosexual lesbian fucking bastards from hell!
....................................
Do you love the pain that you do? And to do a favour... I'm going to do that!

I kicked her so hard failure golden eggs until they are ringing and sing you all the melodies of forward and back pain in an instant! I am the boy Nutcracker who cracks your are crazy nutheads!

And now go to the stupid shit shit cones!
....................................................................
Was it expected?

Oh, Yes, that was!

A movement such as the Mozilla!

So happy that I never upgraded to Firefox 29 and beyond!

Let me tell you something Mozilla developers! If I would be still using your latest crap-ware that really my mission would at this time to find out where you live and then drill him often in Golden's failure until you analyzes and understands how to threaten power users in the correct way!

Without a doubt, has been said before - but may not be repeated enough... You are...

Stupid cones!

**Comment by Lord-of-Darkness** [comment]
Developers and leaders of Mozilla, which are shemales ass!

Someone should hang you up in your own balls and let your dog bite his caricature of a cock!

They should then put it long as needed by the throat until he swallows themselves in suicide!

You Mozilla is... Stupid fucking pussies!
.....................................................
You can lick my penis and my golden eggs and kiss my ass son of a bitch with a personality disorder from the depths of the abyss of hell from the abysses of Mozilla!

Fuck you and your mother you autistic son of a bitch, you will lick Putin eggs, fascist fucking faggot!

The shit out of your dog, your cat and your dirty faggot asshole mountain goat!

Suck cocks Mozilla and together go and suck Google ass full of shit!

Shit in your knee you cock suckers! And that damn Australis Chrome clone of shit! Mozilla gay homosexual lesbian fucking bastards from hell!
....................................
Do you love the pain that you do? And to do a favour... I'm going to do that!

I kicked her so hard failure golden eggs until they are ringing and sing you all the melodies of forward and back pain in an instant! I am the boy Nutcracker who cracks your are crazy nutheads!

And now go to the stupid shit shit cones!
....................................................................
Was it expected?

Oh, Yes, that was!

A movement such as the Mozilla!

So happy that I never upgraded to Firefox 29 and beyond!

Let me tell you something Mozilla developers! If I would be still using your latest crap-ware that really my mission would at this time to find out where you live and then drill him often in Golden's failure until you analyzes and understands how to threaten power users in the correct way!

Without a doubt, has been said before - but may not be repeated enough... You are...

Stupid cones!

**Comment by mehlis** [comment]
not fixed in Fedora 22, Firefox 40.0.3, PDF.js 1.1.444
